CMakeLists.txt

```
@@ -110,7 +117,7 @@ wpilib_config(OPTIONS WITH_WPIMATH WITH_JAVA REQUIRES WITH_WPIUNITS)
 
 set(include_dest include)
 set(java_lib_dest java)
-if(WITH_JAVA OR WITH_JAVA_SOURCE)
+if(WITH_JAVA OR WITH_JAVA_SOURCE OR WITH_WPIUNITS)
```

```suggestion
if(WITH_JAVA OR WITH_JAVA_SOURCE)
```
This change isn't necessary since it's dependent on WITH_JAVA